.github/workflows/_cicd.yml

```diff
@@ -29,32 +25,32 @@ jobs:
   release-dev:
     # if: ${{ false }}  # Uncomment this, and comment out the line below, to disable this workflow path
     if: github.event_name == 'workflow_dispatch'
-    uses: i-dot-ai/redbox/.github/workflows/build-and-release.yml@main
+    uses: i-dot-ai/redbox/.github/workflows/build-and-release.yml@a95ade7abfcf3256cf38c5a5e13344caa0b975c2
```

Comment:
is there anyway we can make this nicer with say a @v1 / @v2 etc? I pressume we'd have to host our own action repo though.

Reply:
Replaced with ref to `main` before merging